api/api_ethereum.go

```diff
@@ -1577,7 +1577,7 @@ func EthDoEstimateGas(ctx context.Context, b Backend, args EthTransactionArgs, b
 	// Binary search the gas requirement, as it may be higher than the amount used
 	var (
 		lo  uint64 = params.TxGas - 1
-		hi  uint64
+		hi  uint64 = 99999999
```

Comment:
could you explain what does this number mean?

Reply:
Oh, it's just a enough big number for highest gas limit. Never mind.

Reply:
@hqjang-pepper How about changing it to params.upperGasLimit or math.MaxUint64?

Reply:
upperGasLimit seems good idea. Thanks

Reply:
Oh, what i wondered is that it looks like hi doesn't need default value. it seems hi are always initialized as  uinit64(*args.Gas) or params.UpperGasLimit